### PR TITLE
Don't hide tagged commits in `dim-bots`

### LINKED ISSUE #4318: `dim-bots` hides GitHub Actions commits
https://github.com/fregante/github-url-detection/commits/master?after=e58ce98ffd6a45822d035598eda8f699e1839f06+1&branch=master

Screen Shot

Notice the 4.11.0 commit being dimmed.

### PR DESCRIPTION
**Pull request by cheap-glitch**
Fixes #4318 

## Test URLs
https://github.com/fregante/github-url-detection/commits/master?after=e58ce98ffd6a45822d035598eda8f699e1839f06+1&branch=master

## Screenshot

<table>
<tr>
	<th>Before
	<td>

image
<tr>
	<th>After
	<td>

image
</table>

The tags are lazy-loaded. Not sure using `oneMutation` is the best approach though.

### CODE REVIEW

File: source/features/dim-bots.tsx [hunk 1]
```diff
@@ -15,13 +16,25 @@ const botSelector = [
...
+async function init(): Promise<void> {
 	for (const bot of select.all(botSelector)) {
 		// Exclude co-authored commits
 		if (select.all('a', bot.parentElement!).every(link => link.matches(botSelector))) {
 			bot.closest('.commit, .Box-row')!.classList.add('rgh-dim-bot');
 		}
 	}
+
+	if (!tagsLoaded()) {
+		await oneMutation(document.body, {subtree: true, childList: true, filter: tagsLoaded});
```
Comment (cheap-glitch) [inline source/features/dim-bots.tsx:32]: Maybe we need some kind of timeout here? Since the promise can hang forever on commit lists with no tags.

**Comment by fregante** [comment]
No, here you just need to add a class to the .rgh-dim-bot element and then just handle it via CSS (:not(tagged))

We should almost never need to wait for mutations since, at worst, we can fire custom events and listen to them. One example is the “markdown source” feature

**Comment by cheap-glitch** [comment]
> We should almost never need to wait for mutations

I'm confused :thinking: Since the tags are lazy-loaded, don't we need some kind of listener? Or should this be a simple `setTimeout`?

**Comment by cheap-glitch** [comment]
Closing this as I won't have time to work on it.

**Comment by fregante** [comment]
> Since the tags are lazy-loaded

The tags are not lazy loaded: Refined GitHub is adding the tags. The difference is that since we add them, we also control the code that adds them. In short, the `.rgh-tagged` class should be added by `tags-on-commits-list`, not here.

**Pull request closed without merging**